simple/msg_pingpong.c

```diff
@@ -548,9 +548,7 @@ int main(int argc, char **argv)
 	while ((op = getopt(argc, argv, "vhg:" CS_OPTS INFO_OPTS)) != -1) {
 		switch (op) {
 		case 'v':
-			printf("%s: %s\n", argv[0], PACKAGE_VERSION);
-			printf("libfabric: %s\n", fi_tostr("1", FI_TYPE_VERSION));
-			printf("libfabric api: %d.%d\n", FI_MAJOR(FT_FIVERSION), FI_MINOR(FT_FIVERSION));
+			ft_version(argv[0]);
```

Comment:
> I thought it would be useful for debugging, maybe not?

I think it's useful for one app to output this (fi_info), but unnecessary for all to support.  The data being displayed isn't related to the test app, but the library itself.